_search-plugins/knn/api.md

```diff
@@ -175,6 +175,48 @@ For the warmup operation to function properly, follow these best practices:
 
 * Don't index any documents that you want to load into the cache. Writing new information to segments prevents the warmup API operation from loading the native library indexes until they're searchable. This means that you would have to run the warmup operation again after indexing finishes.
 
+## k-NN Clear Cache
+Introduced 2.14
+{: .label .label-purple }
+
+During approximate k-Nearest Neighbor (k-NN) search or warmup operation, the native library indices(for `nmslib` and `faiss` engines) are loaded into native memory. Currently, the possible ways to evict an index from cache or native memory is by either deleting the index or by setting a 
```

Comment:
Can we add a bold note that this has no effect for lucene engine? Just want to call out explicitly.

Reply:
updated